templates/crds/common-template-bundles.crd.yaml.in

```diff
@@ -0,0 +1,15 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Should put the definition in a `{{if .Converged}}` so it doesn't end up in the standard manifests. Same for the others that only really make sense in the converged case.